packages/astro/components/Picture.astro

```diff
@@ -24,16 +24,14 @@ const optimizedImages: GetImageResult[] = await Promise.all(
 	)
 );
 
-const fallbackFormat =
-	props.fallbackFormat ?? isESMImportedImage(props.src)
-		? ['svg', 'gif'].includes(props.src.format)
-			? props.src.format
-			: 'png'
-		: 'png';
+let resultFallbackFormat = fallbackFormat ?? 'png';
+if (isESMImportedImage(props.src) && ['gif', 'svg'].includes(props.src.format)) {
```

Comment:
nit: `['gif', 'svg']` could be moved into a top-level variable

Reply:
I'm gonna agree here, it took me a second to understand what they did while investigating the issue on the previous code, and it's the same here. A tiny bit better, but a separate variable with a descriptive name would go along way.

Reply:
Sure, we can do that